pkg/instrumentation/dotnet_test.go

```diff
@@ -434,13 +443,15 @@ func TestInjectDotNetSDK(t *testing.T) {
 					},
 				},
 			},
+			sdkInjectionSkipped: true,
 		},
 	}
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			pod := injectDotNetSDK(logr.Discard(), test.DotNet, test.pod, 0)
+			pod, sdkInjectionSkipped := injectDotNetSDK(logr.Discard(), test.DotNet, test.pod, 0)
```

Comment:
Can you change the contract to the positive value? `sdkInjectionSkipped` -> `sdkInjected`. It will requires changes in all places.

BTW current scenario looks like isDisabled=true, which is usually hard to understand and maintain.

Reply:
Done.

Reply:
Feel free to resolve.